build-tools/packages/build-cli/src/library/commands/index.ts

```diff
@@ -9,3 +9,8 @@ export {
 	GenerateEntrypointsCommand,
 	getGenerateEntrypointsOutput,
 } from "./generateEntrypoints.js";
+export {
```

Comment:
No need to export here - just import from the submodules directly and suppress any lint error. See AB#8118